DOCS-3024 Update Event migration guide for simplified process (#12933)
DataDog/documentation: content/en/events/guides/migrating_to_new_events_features.md

## What this means for your organization

* New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.

* Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).

This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).

## Migration process

### Datadog migrates monitors and dashboards configured through the UI
<div class="alert alert-info">
You do not need to take any action for dashboards or monitors created through Datadog.
</div>

The changes to Event query syntax affect two Datadog features: monitors and dashboard widgets.

Datadog migrates monitors that query Events by doing the following:

> - For each legacy Event monitor, create a duplicate that uses the new query syntax. Enable the new Event monitor.
> - Add the prefix "legacy" to the legacy Event monitor, and mute it.
> - Update all SLOs and monitor downtimes that reference the legacy Event monitor to point to the new monitor.

Reviewer: ```suggestion
- For each legacy Event monitor, creates a duplicate that uses the new query syntax. Enables the new monitor.
- Adds the prefix "legacy" to the legacy Event monitor, and mutes it.
- Updates all SLOs and monitor downtimes that reference the legacy Event monitor to point to the new monitor.
```

> When viewing monitors, you can filter which of the two monitor types you would like to see using the **Event Migration** facet. This facet has available values "Event Monitors V1" and "Event Monitors V2."

Reviewer: ```suggestion
When viewing monitors, you can filter which of the two monitor types you would like to see using the **Event Migration** facet. This facet has available values `Event Monitors V1` and `Event Monitors V2`.
```

Datadog directly migrates dashboard widgets that use Event queries. These widgets include Event Stream widgets, Event Timeline widgets, and Event overlays in other widgets. Datadog does not create any new dashboards or widgets.

### Migrate data configured externally (Terraform, API)

<div class="alert alert-warning">
You must take action to migrate Datadog configuration you manage with Terraform or scripts.
</div>

If you manage your Datadog configuration with external, API-based scripts or tools such as Terraform, update your dashboard and monitor definitions in your configuration file. Otherwise, the old query syntax is applied again at the next deployment and overwrites updated queries. Review the syntax changes described below and use the [migration script][3] to see what converted queries look like. [Contact Support][4] for assistance at this stage.

## Post migration

### Access

You can access the new Explorer and other features through the Events menu in the main navigation, alongside the existing Stream.

{{< img src="events/guides/navigation.png" alt="Events navigation" style="width:100%;" >}}

> ### User impact

Reviewer: ```suggestion
### Impact
```

The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.

## What this means for your organization

* New events (after migration) are visible only in the [Events Explorer][2] or dashboard widgets and event overlays.

* Older events (before migration) continue to be visible in the [Event Stream][2] for the remainder of their retention period (13 months).

This guide describes how to migrate your organization’s event monitors and dashboard widgets so that you benefit from the new Events features. Following this guide ensures that you are ready when Datadog retires the legacy event stream and storage. This guide also provides [a detailed description of what's changed](#details-of-the-changes).

## Migration process

### Datadog migrates data stored in Datadog servers
<div class="alert alert-info">

> You do not need to take any action for configuration stored directly in Datadog.

Reviewer 2: You do not need to take any action for any dashboards or monitors created via Datadog.

</div>

> Datadog is responsible for migrating data stored in its servers. You do not need to take any action for configuration stored directly in Datadog. The changes to Event query syntax affect two Datadog features: monitors and dashboard widgets.

Reviewer 2: same as above. maybe remove this paragraph.

Datadog migrates monitors that query Events by doing the following:
- For each legacy Event monitor, create a duplicate that uses the new query syntax. Enable the new Event monitor.
- Add the prefix "legacy" to the legacy Event monitor, and mute it.
- Update all SLOs and monitor downtimes that reference the legacy Event monitor to point to the new monitor.

When viewing monitors, you can filter which of the two monitor types you would like to see using the **Event Migration** facet. This facet has available values "Event Monitors V1" and "Event Monitors V2."

> Datadog directly migrates dashboard widgets that use Event queries. Datadog does not create any new dashboards or widgets.

Reviewer 2: Perhaps add more details after the first sentence: This includes Event Stream, Event Timeline widgets and event overlays in other widgets.

---

<div class="alert alert-warning">
  Use this guide to migrate from the legacy Events Stream to the Events Explorer. The Events Explorer is being automatically rolled out to all customers in Q1 and Q2 2022. If you see the <strong>Explorer</strong> option under <strong>Events</strong> in the left navigation bar, you have the new features. For legacy events information, see the <a href="/events/stream/">Event Stream</a>.</div>

The Events Explorer replaces the Event Stream, one of Datadog's oldest features. The new Event query syntax requires migration of Event monitors, dashboard widgets, and any other features that query Events. 

> The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.

Reviewer 3: To add timeline in intro with sunset date (final date still TBD)

> ## What this means for your organization

Reviewer 3: This will slightly change (details still TBD - please ask #event-migration if no updates in the next days)

## Migration process

### Migrate data configured externally (Terraform, API)
If you manage your Datadog configuration with external, API-based scripts or tools such as Terraform, update your dashboard and monitor definitions in your configuration file. Otherwise the old query syntax is applied again at the next deployment and overwrites updated queries. Review the syntax changes described below and use the [migration script][3] to see what converted queries look like. [Contact Support][4] for assistance at this stage.

> ### Datadog migrates data stored in Datadog servers

Reviewer 3: I'd switch the 2 sections are be more precise.
Something along the line of:
1. Monitor and dashboards definitions stored on Datadog servers are automatically updated (no action required)
2. Definitions stored externally (Terraform or API-based configuration tools must update) must be updated (your action is required)

> Datadog is responsible for migrating data stored in its servers. You do not need to take any action for configuration stored directly in Datadog. Datadog automatically migrates Event data contained in monitors and dashboard widgets, including the event stream, timeline, and overlays.

Reviewer 3: to precise:

1) legacy event monitors will be duplicated:
- new monitors based on the new query will be created and automatically enabled.
- existing monitors will be muted. A "(legacy)" prefix will be added to distinguish them.
You can filter the 2 monitor types using the "Event Migration" facet in the monitors list: https://a.cl.ly/rRuNAgm1

SLO and downtimes links will also be updated to point to the new monitors.

2) dashboard widgets using event queries will directly updated.
No new dashboard will be created
